chia/cmds/cmds_util.py

```diff
@@ -71,7 +73,7 @@ async def get_any_service_client(
         # Click already checks this, so this should never happen
         raise ValueError(f"Invalid node type: {node_type}")
     # load variables from config file
-    config = load_config(root_path, "config.yaml")
+    config = load_config(root_path, "config.yaml", fill_missing_services=True)
```

Comment:
This behavior was very specifically not applied in general.  Was this carefully thought through in terms of the effect on the config file getting written or anything else?

Reply:
No, i'm going to quickly change it.

Reply:
@altendky https://github.com/Chia-Network/chia-blockchain/pull/13669